### PR TITLE
Project creation

### PR DESCRIPTION
**Pull request by mgamini**
tenor-267788552

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by reset** [comment]
Closes #3237

File: components/builder-web/app/actions/projects.ts [hunk 1]
```diff
@@ -13,12 +13,28 @@
 // limitations under the License.
 
 import * as fakeApi from "../fakeApi";
-import {Observable} from "rxjs";
-import {BuilderApiClient} from "../BuilderApiClient";
-import {addNotification} from "./notifications";
-import {DANGER, INFO, SUCCESS, WARNING} from "./notifications";
-import {requestRoute, resetRedirectRoute} from "./router";
-import {packageString} from "../util";
+import {
```
Comment (elliott-davis) [inline components/builder-web/app/actions/projects.ts]: I think this is the prefered style: https://angular.io/guide/styleguide#import-line-spacing. We should try to get the linter to enforce that later.

File: components/builder-web/app/package/package-settings/package-settings.component.ts [hunk 2]
```diff
@@ -18,6 +18,14 @@ export class PackageSettingsComponent implements OnDestroy {
         });
     }
 
+    planSelectSubmit(plan) {
+        console.log(plan);
+    }
+
+    planSelectCancel() {
+        console.log("on cancel");
```
Comment (elliott-davis) [inline components/builder-web/app/package/package-settings/package-settings.component.ts]: This should be a `console.debug()`

File: components/builder-web/app/project-create-page/ProjectCreatePageComponent.ts [hunk 3]
```diff
@@ -59,6 +58,11 @@ export class ProjectCreatePageComponent implements OnInit, OnDestroy {
         this.sub.unsubscribe();
     }
 
+    handleSubmit(selection) {
+        console.log(selection);
```
Comment (elliott-davis) [inline components/builder-web/app/project-create-page/ProjectCreatePageComponent.ts]: `console.debug` here as well

File: components/builder-web/app/project-settings-page/ProjectSettingsPageComponent.ts [hunk 4]
```diff
@@ -34,10 +34,10 @@ import {Record} from "immutable";
           </p>
       </div>
       <div class="page-body">
-        <hab-project-info [project]="project" [ownerAndRepo]="ownerAndRepo"></hab-project-info>
       </div>
     </div>
     `
+    // <hab-project-info [project]="project" [ownerAndRepo]="ownerAndRepo"></hab-project-info>
```
Comment (elliott-davis) [inline components/builder-web/app/project-settings-page/ProjectSettingsPageComponent.ts]: Should this be removed?

File: components/builder-web/app/shared/github-repo-picker/GitHubRepoPickerComponent.ts [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+import { List, Map, OrderedSet } from "immutable";
+import { AppStore } from "../../AppStore";
+import { GitHubRepo } from "../../github/repo/shared/github-repo.model";
+import {fetchGitHubOrgs, fetchGitHubRepos,
+  onGitHubOrgSelect, setSelectedGitHubOrg, resetRedirectRoute} from "../../actions/index";
+
+@Component({
+  selector: "hab-github-repo-picker",
+  template: `
+  <div class="hab-github-repo-picker">
```
Comment (elliott-davis) [inline components/builder-web/app/shared/github-repo-picker/GitHubRepoPickerComponent.ts]: Could you move this file into it's own html file? https://angular.io/guide/styleguide#extract-templates-and-styles-to-their-own-files

File: components/builder-web/npm-shrinkwrap.json [hunk 6]
```diff
@@ -1,1735 +0,0 @@
-{
```
[the remark this answers is not in the document]
Comment (mgamini) [reply components/builder-web/npm-shrinkwrap.json]: npm shrinkwrap only exists in newer versions of node than we're using, right?

File: components/builder-web/app/actions/projects.ts (hunk 1 above)
Comment (mgamini) [reply components/builder-web/app/actions/projects.ts]: ah sorry, ran an auto beautifier

**Comment by mgamini** [comment]
@elliott-davis done!

File: components/builder-web/npm-shrinkwrap.json (hunk 6 above)
[the remark this answers is not in the document]
Comment (elliott-davis) [reply components/builder-web/npm-shrinkwrap.json]: no it's there in ours as well

**Comment by elliott-davis** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/278098651).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/278098651).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**